tests/test_table.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+import py_osrm
+import constants
+
+data_path = constants.data_path
+mld_data_path = constants.mld_data_path
+three_test_coordinates = constants.three_test_coordinates
+two_test_coordinates = constants.two_test_coordinates
+
+class TestTable:
+    osrm = py_osrm.OSRM(
+        storage_config = data_path, 
+        use_shared_memory = False
+    )
+
+    def test_table_annotations(self):
+        table_params = py_osrm.TableParameters(
+            coordinates = [three_test_coordinates[0], three_test_coordinates[1]],
+            annotations = ["distance"]
+        )
+        res = self.osrm.Table(table_params)
+        assert(res["distances"])
+        assert(not res["durations"])
+
+        table_params = py_osrm.TableParameters(
+            coordinates = [three_test_coordinates[0], three_test_coordinates[1]],
+            annotations = ["duration"]
+        )
+        res = self.osrm.Table(table_params)
+        assert(res["durations"])
+        assert(not res["distances"])
+
+        table_params = py_osrm.TableParameters(
+            coordinates = [three_test_coordinates[0], three_test_coordinates[1]],
+            annotations = ["duration", "distance"]
+        )
+        res = self.osrm.Table(table_params)
+        assert(res["durations"])
+        assert(res["distances"])
+
+        table_params = py_osrm.TableParameters(
+            coordinates = [three_test_coordinates[0], three_test_coordinates[1]]
+        )
+        res = self.osrm.Table(table_params)
+        assert(res["durations"])
+        assert(not res["distances"])
+
+    def test_table_snapping(self):
+        table_params = py_osrm.TableParameters(
+            coordinates = [three_test_coordinates[0], three_test_coordinates[1]],
+            snapping = "any"
+        )
+        res = self.osrm.Table(table_params)
+        assert(res["durations"])
+
+    # def test_table_annotation(self):
+    #     tables = ["distance", "duration"]
```

Comment:
did you try these again? if they currently don't work, let's shelve it for after the GHA workflow

Reply:
Currently, no. I'll take another look after GHA workflow!